addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -581,6 +581,29 @@ def _l10n_in_edi_generate_invoice_json(self, invoice):
                 json_payload["ExpDtls"].update({
                     "Port": invoice.l10n_in_shipping_port_code_id.code
                 })
+            json_valdtls = json_payload['ValDtls']
+            base_and_tax_amount = tax_details.get("base_amount") + tax_details.get("tax_amount")
+            # For Export If with payment of Tax then we need to include Tax in Total Invoice Value
+            if json_payload['TranDtls']['SupTyp'] == 'EXPWP' and json_valdtls['AssVal'] == base_and_tax_amount:
```

Comment:
on line 527:
`"AssVal": self._l10n_in_round_value(tax_details.get("base_amount")),`

on 587: 
the condition is `json_valdtls['AssVal'] == tax_details.get("base_amount") + tax_details.get("tax_amount")`

So the tax_amount is actually zero... yet we are including taxes due to the screenshot you shared with me.

Could I ask to please mention that in the comment here
`# For Export, we include tax in Total Invoice Value despite there being no tax on the document`

You could also reference the link you shared with me, or at least, mention it on the task.